internal/rbd/replicationcontrollerserver.go

```diff
@@ -368,8 +368,26 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 	ready := false
 	state := imageMirroringState(mirrorStatus.State)
 	if state == upAndStopped || state == upAndReplaying {
+		// Make sure the peer site image state is up and stopped
 		ready = true
+		for _, s := range mirrorStatus.PeerSites {
+			if imageMirroringState(s.State) != upAndStopped {
+				util.DebugLog(ctx, "peer site name=%s mirroring state=%s, description=%s and lastUpdate=%s", s.SiteName, s.State, s.Description, s.LastUpdate)
```

Comment:
I always think of `DebugLog` for developers, and `DefaultLog` or `InfoLog` and higher for users. Do you want to assist users with troubleshooting potential mirroring issues?

Reply:
Moved to InfoLog